src/quacc/recipes/espresso/phonons.py

```diff
@@ -463,3 +480,135 @@ def _grid_phonon_subflow(
     )
 
     return _ph_recover_subflow(grid_results)
+
+
+@job
+def dvscf_q2r_job(
+    copy_files: (
+        SourceDirectory | list[SourceDirectory] | dict[SourceDirectory, Filenames]
+    ),
+    parallel_info: dict[str] | None = None,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic dvscf_q2r calculation. It should allow you to
```

Comment:
Not immediately clear based on this sentence what a dvscf_q2r calculation actually is.